source/WB-CAN-Firmware.c

```diff
@@ -41,15 +41,20 @@
 #define TC_I2C3_CLK_FREQ CLOCK_GetFreq((I2C3_CLK_SRC))
 #define TC_I2C3 ((I2C_Type *)TC_I2C3_BASE)
 
+#define BUFFERSIZE 10
 /*******************************************************************************
  * Task Prototypes
  ******************************************************************************/
+static void mainTask(void *pv);
+
 static void blinkTask(void *pv);
 static void testTask(void *pv);
 static void actuatorTask(void *pv);
 static void ControlTask(void *pv);
 static void tcTask(void *pv);
 
+static void RTTreceive(void *pv);
```

Comment:
We can probably remove these empty lines

```suggestion
static void blinkTask(void *pv);
static void testTask(void *pv);
static void actuatorTask(void *pv);
static void ControlTask(void *pv);
static void tcTask(void *pv);
static void RTTreceive(void *pv);
```